src/iop/lens.cc

```diff
@@ -1143,7 +1164,19 @@ void commit_params(struct dt_iop_module_t *self, dt_iop_params_t *p1, dt_dev_pix
       *d->lens = *lens[0];
       if(p->tca_override)
       {
-#ifdef LF_0395
+
+#ifdef LF_2000
+          const dt_image_t *img = &(self->dev->image_storage);
+
+          d->custom_tca.Model = LF_TCA_MODEL_LINEAR;
+          d->custom_tca.Focal = p->focal;
+          d->custom_tca.Terms[0] = p->tca_r;
+          d->custom_tca.Terms[1] = p->tca_b;
+
+          d->custom_tca.CalibAttr.CropFactor = d->crop;
+          d->custom_tca.CalibAttr.AspectRatio = (float)img->width / (float)img->height;
+
+#elif LF_0395
```

Comment:
Another duplicated block, though using a different approach to initialization.  Did you forget to init .CenterX and .CenterY?

Reply:
CalibAttr is of a type: struct lfLensCalibAttributes
Which only has two attributes in the new interface:
    /** Crop factor at which calibration measurements were taken.  Must be defined. */
    float CropFactor;
    /** Aspect ratio of the images used for calibration measurements. */
    float AspectRatio;